consensus/src/execution_ctx.rs

```diff
@@ -225,24 +226,31 @@ impl<'a, DB: Database, T: Operations + 'static> ExecutionCtx<'a, DB, T> {
             future_msgs,
             provisioners,
             round_update,
+            iteration,
             step,
             _executor: executor,
             sv_registry,
             quorum_sender,
         }
     }
 
+    pub fn total_step(&self) -> u16 {
```

Comment:
total_step sounds misleading as we  have `fn get_step(&self) -> u16 ` in MessageTrait that returns the absolute step number too. 

Maybe we can use a naming convention:
`get_step()/step()` ->  returns absolute/total step number in all traits/structs
`get_step_name()/step_name()` ->  returns step name only

step_name: StepName
step: u8

What do you think?

Reply:
Good idea!